azure-pipelines.yml

```diff
@@ -64,19 +83,23 @@ jobs:
 # Debug build
 #
 
-- template: eng/platform-matrix.yml
-  parameters:
-    jobTemplate: build-job.yml
-    buildConfig: debug
+# Official build is release only.
```

Comment:
This looks reversed -- I think it only enables debug for official builds. Ditto for the others below.

Reply:
I am confused, TeamProject != public && build.Reason != PullRequest seems correct. Also scheduled jobs are release, see https://dnceng.visualstudio.com/internal/_build/results?buildId=50765.

Reply:
TeamProject != public && buildReason != PullRequest equates to official build (In other words, TeamProect == internal and buildReason == official). This is adding the debug builds when that's true, but not otherwise.

Reply:
I believe this should be addressed.